### PR TITLE
Registration Zip and Encrypted Passwords

### PR DESCRIPTION
**Pull request by Jody-Whitis**
- Added Zipcode to form.
- Added encryption on passwords with Unit Test.
- Added Generic Session Login Function.
- Updated CSS on Login, Register, and _Layout.
- Resolved merge conflicts.
- Updated _Layout background hieght.
image

### CODE REVIEW

File: QuickFixers/Controllers/LoginController.cs [hunk 1]
```diff
@@ -50,24 +50,21 @@ public ActionResult Index(HomeViewModel homeViewModelPost)
 
                 // Add parameter names and values to dictionary
                 spParameters.Add("@Email", homeViewModelPost.Email);
-                spParameters.Add("@UserPassword", homeViewModelPost.UserPassword);
+                spParameters.Add("@UserPassword", homeViewModelPost.UserPassword.ToEncriptedString());
```
Comment (BSaenz12) [inline QuickFixers/Controllers/LoginController.cs]: Please rename function to `ToEncryptedString`

File: QuickFixers.Data/Utilities/Validations.cs [hunk 2]
```diff
@@ -13,5 +14,19 @@ public static string ToSqlSafe(string sqlField)
         {
             return sqlField.Replace("'", "''");
         }
+
+        public static string ToEncriptedString(this string passwordText)
```
Comment (BSaenz12) [inline QuickFixers.Data/Utilities/Validations.cs]: Rename function

File: QuickFixers/Controllers/LoginController.cs (hunk 1 above)
Comment (Jody-Whitis) [reply QuickFixers/Controllers/LoginController.cs]: updated function name

**Comment by BSaenz12** [comment]
Good job!

**Pull request merged**